### PR TITLE
Bekir London 10 FullStackProject

### PR DESCRIPTION
**Pull request by BekirKuscu**

### CODE REVIEW

**Comment by BEKIRKSU** [comment]
All 300 levels done.

**Review by ShadiSafaee — COMMENTED** [review]
Hi @BEKIRKSU - could you add node_modules to .gitignore file? You can follow [this ](https://medium.com/@rspesc31/forgot-to-add-node-modules-to-gitignore-solved-6c5b7a444c65)to find out how. Currently, I'm unable to view your code because your node_modules have consumed a significant amount of storage in your pull request.

File: client/src/AddVideo.jsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+        console.log("Server response:", response.data); // Debugging line
+  
+        // If the video is added successfully on the server, update the client-side state
+        if (response.status === 201) {
+          onAdd(response.data); // Assuming response.data contains the new video data
+          setTitle('');
+          setUrl('');
+  
+          // Reload the page
+          window.location.reload();
```
Comment (illicitonion) [inline client/src/AddVideo.jsx:33]: You shouldn't need to reload pages in react apps - if you set new state, react will re-render the page for you if it needs it.

File: server/config/db.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+const {Pool} = require("pg");
+const fs = require("fs");
+
+
```
Comment (illicitonion) [inline server/config/db.js:4]: You shouldn't add your password to source control - I'd recommend removing this, using a .env file, and changing your password.

File: server/server.js [hunk 3]
```diff
@@ -1,15 +1,93 @@
...
+    console.error("Error adding video to the database", error);
+    res.status(500).json({ error: "Internal server error" });
+  }
+});
+
+
+
+app.get("/:id", (req, res) => {
+  const id = parseInt(req.params.id);
+  const video = videos.find((video) => video.id === id);
```
Comment (illicitonion) [inline server/server.js:69]: It looks like this didn't get updated to use the database? Same as the delete, below?

File: client/public/index.html [hunk 4]
```diff
@@ -10,42 +10,58 @@
...
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
 
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
+    <script>
```
Comment (illicitonion) [inline client/public/index.html:30]: What's the deal with this script here?

**Review by illicitonion — COMMENTED** [review]
I removed your node_modules so I could review this code - it's generally looking good, but it looks like you have a few pieces of leftover code you maybe don't need any more?

**Pull request closed without merging**